src/components/grid-aware/BlockQuoteBlock/BlockQuoteBlock.js

```diff
@@ -0,0 +1,49 @@
+import PropTypes from "prop-types";
+import { CarouselProvider, Slider, Slide, DotGroup } from "pure-react-carousel";
+import React from "react";
+
+import "pure-react-carousel/dist/react-carousel.es.css";
+import s from "./BlockQuoteBlock.module.css";
+
+const BlockQuoteBlock = ({ quotes }) => {
+  const Carousel = () => (
+    <CarouselProvider totalSlides={quotes.length} isIntrinsicHeight>
+      <Slider>
+        {quotes.map((dataObj, index) => (
```

Comment:
Nit: But I usually like destructuring the `map()` argument into the individual properties you want to use:

```suggestion
        {quotes.map(({ quote, attribute }, index) => (
```

When you do that, you no longer have to access the properties on the `dataObject` variable, since you have individual variables for each property.

Reply:
👍 Destructured.